addon/utils/display-type.js

```diff
@@ -206,6 +207,10 @@ registerComponentsForDisplayType([
     displayType: 'http://lblod.data.gift/display-types/property-group',
     edit: PropertyGroupComponent,
   },
+  {
+    displayType: 'http://lblod.data.gift/display-types/section',
+    edit: SectionComponent,
+  },
```

Comment:
Hmmmm, I think this isn't needed and the property-group version is probably a mistake I made in the past. Do we use these display types anywhere? I think we retrieve all the sections with a helper in the relevant components?

Reply:
seems you are right, the section component is handled explicitly where the component type for other components is determined dynamically if it's not a section or listing